### PR TITLE
Charged system tutorial script: undefined variables

### PR DESCRIPTION
**Pull request by reinaual**

### CODE REVIEW

**Review by jngrad — APPROVED** [review]

File: doc/tutorials/02-charged_system/scripts/nacl_units_confined_vis.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 #l_bjerrum = 0.885^2 * e^2/(4*pi*epsilon_0*k_B*T)
 l_bjerrum = 130878.0 / temp
 Vz = 0  # potential difference between the electrodes
-Ez = 364.5  # conversion from potential to electrical field
+Vz_to_Ez = 364.5  # conversion from potential to electrical field
```
Comment (KaiSzuttor) [inline doc/tutorials/02-charged_system/scripts/nacl_units_confined_vis.py:47]: I think we should use capital letter names for all constants https://www.python.org/dev/peps/pep-0008/#id48

**Review by KaiSzuttor — DISMISSED** [review]

File: doc/tutorials/02-charged_system/scripts/nacl_units_confined_vis.py (hunk 1 above)
Comment (jngrad) [reply doc/tutorials/02-charged_system/scripts/nacl_units_confined_vis.py:47]: Deferred to espresso 5, moved to a dedicated issue.

**Pull request merged**